soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4157,6 +4175,8 @@ void DrawRandoEditor(bool& open) {
                 UIWidgets::InsertHelpHoverText("Talking to Ganondorf in his boss room will tell you the location of the Light Arrows. If this option is enabled and Ganondorf is reachable without Light Arrows, Gossip Stones will never hint the Light Arrows.");
                 UIWidgets::PaddedEnhancementCheckbox("Dampe's Diary (Hookshot)", "gRandomizeDampeHint", true, false);
                 UIWidgets::InsertHelpHoverText("Reading the diary of Dampé the gravekeeper as adult will tell you the location of one of the Hookshots.");
+                UIWidgets::PaddedEnhancementCheckbox("Greg the Green Rupee", "gRandomizeGregHint", true, false);
+                UIWidgets::InsertHelpHoverText("Talking to the chest game owner will tell you the location of Greg the Green Rupee.");
```

Comment:
Should we mention something like "Talking to chest game owner _after buying a key_?"

Reply:
updated!